src/software/ai/hl/stp/tactic/attacker/attacker_fsm.h

```diff
@@ -103,21 +144,22 @@ struct AttackerFSM
          *
          * @return if the ball should be kicked
          */
+        // TODO: revisit this, we shouldn't "panic chip" unless we're completely boxed in!
```

Comment:
todo to revisit

Reply:
This is out of scope for the PR, will remove TODO

Reply:
revisited

Reply:
the todo is still there

Reply:
deleted the todo